README.md

```diff
@@ -137,6 +137,32 @@ const ExampleComponent = () => {
 };
 ```
 
+## Dynamic Height Control
+
+You can dynamically control the maximum height of your iframe when embedded in Workshop using the `setAutoMaxHeight` function that's returned as part of the workshop context. This allows your application to adjust its container height based on content changes, user interactions, or any other factors.
+
+**Important Note:** The `setAutoMaxHeight` function only works when the Workshop widget's height is set to "auto (max)" in the Workshop widget settings.
```

Comment:
say the iframe has a height of auto max 500px in Workshop. We should say some thing about what happens if the iframe tries to `.setAutoMaxHeight(600)` does it fail? get ignored? get set the the total possible max of 500 if it exceeds it?